### ISSUE TITLE
Bug: Low number of scans

### ISSUE THREAD

**Issue by drwetter** [sullo/nikto#700]
### Expected behavior

If I add -T/-Tuning options it should scan more, not less

### Actual behavior + steps to reproduce

```
myprompt% $PATH/nikto.git/program/nikto.pl   -useragent <..>  -host <..> -Tuning 1 
[..]
+ 2463  requests: 0 error(s) and 8 item(s) reported on remote host
+ End Time:           2020-09-19 18:57:52 (GMT2) (12 seconds)
---------------------------------------------------------------------------
+ 1 host(s) tested
myprompt% myprompt% $PATH/nikto.git/program/nikto.pl   -useragent <..>  -host <..> -Tuning 12
[..]
+ 565 requests: 0 error(s) and 8 item(s) reported on remote host
+ End Time:           2020-09-19 19:02:51 (GMT2) (3 seconds)
---------------------------------------------------------------------------
+ 1 host(s) tested
myprompt% myprompt% $PATH/nikto.git/program/nikto.pl   -useragent <..>  -host <..> -Tuning 123
[..]
+ 530 requests: 0 error(s) and 8 item(s) reported on remote host
+ End Time:           2020-09-19 19:03:30 (GMT2) (2 seconds)
+ 787 requests: 0 error(s) and 8 item(s) reported on remote host
+ End Time:           2020-09-19 19:03:41 (GMT2) (3 seconds)
---------------------------------------------------------------------------
+ 1 host(s) tested
myprompt% myprompt% $PATH/nikto.git/program/nikto.pl   -useragent <..>  -host <..> -Tuning 23
+ 787 requests: 0 error(s) and 8 item(s) reported on remote host
+ End Time:           2020-09-19 19:03:41 (GMT2) (3 seconds)
---------------------------------------------------------------------------
+ 1 host(s) tested
```

### Nikto version

_freshly pulled from github_

----

I used always the negated version before (-T x<numbers> and just to be more clear I wanted to change that. Do I or nikto miss something?

**Comment by sullo** [comment]
By default all tests are run. 

If you use "-Tuning 1" it will _only_ run type 1 tests and not the others.

If you use "-Tuning x1" it will run all test types _except_ type 1

If I'm reading you right it sounds like you just want to remove -Tuning altogether to run all tests?

**Comment by sullo** [comment]
This has been fixed (I believe) in the master-2.5.0 branch. Please checkout that branch and test, and reply here with confirmation it is working. 

Note that if you run something like `-T 12` the request total will **not** be the same as if you run `-T 1` and `-T 2` separately, as some tests may match both and thus duplicated in the individual flag runs.

**Comment by iasdeoupxe** [comment]
> It'll take a while for me. @konstantingoretzki ?

Was confirmed already in https://github.com/sullo/nikto/issues/719#issuecomment-835847356
